src/QMCWaveFunctions/IonOrbital.cpp

```diff
@@ -195,19 +195,18 @@ IonOrbital::evaluateLogAndStore(ParticleSet& P,
 }
 
 /** equivalent to evalaute with additional data management */
-IonOrbital::RealType
-IonOrbital::registerData(ParticleSet& P, PooledData<RealType>& buf)
+void
+IonOrbital::registerData(ParticleSet& P, WFBufferType& buf)
```

Comment:
Do we have any tests to indicate the IonOrbital is working pre/post modification? When is it needed?

Reply:
No idea even pre modification.